src/legacy/core_plugins/tile_map/public/components/wms_internal_options.tsx

```diff
@@ -0,0 +1,225 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import { EuiLink, EuiSpacer, EuiText, EuiScreenReaderOnly } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import { VisOptionsSetValue } from 'ui/vis/editors/default';
+import { TextInputOption } from '../../../kbn_vislib_vis_types/public/controls/text_input';
+
+export interface TmsLayer {
+  id: string;
+  origin: string;
+  minZoom: string;
+  maxZoom: number;
+  attribution: string;
+}
+
+interface WmsOptions {
+  enabled: boolean;
+  url: string;
+  selectedTmsLayer: TmsLayer;
+  options: {
+    attribution: string;
+    format: string;
+    layers: string;
+    styles: string;
+    version: string;
+  };
+}
+
+interface WmsInternalOptions {
+  wms: WmsOptions;
+  setValue: VisOptionsSetValue;
+}
+
+function WmsInternalOptions({ wms, setValue }: WmsInternalOptions) {
+  const wmsLink = (
+    <EuiLink href="http://www.opengeospatial.org/standards/wms" target="_blank">
+      <FormattedMessage id="tileMap.wmsOptions.wmsLinkText" defaultMessage="here" />
+    </EuiLink>
+  );
+  const footnoteText = (
+    <>
+      <span aria-hidden="true">*</span>
+      <FormattedMessage
+        id="tileMap.wmsOptions.mapLoadFailDescription"
+        defaultMessage="If this parameter is incorrect, maps will fail to load."
+      />
+    </>
+  );
+  const footnote = (
+    <EuiScreenReaderOnly>
+      <p>{footnoteText}</p>
+    </EuiScreenReaderOnly>
+  );
+
+  const setOptions: VisOptionsSetValue = (paramName, value) =>
+    setValue('options', {
+      ...wms.options,
+      [paramName]: value,
+    });
+
+  return (
+    <>
+      <EuiSpacer size="xs" />
+      <EuiText size="xs">
+        <FormattedMessage
+          id="tileMap.wmsOptions.wmsDescription"
+          defaultMessage="WMS is an OGC standard for map image services. For more information, go {wmsLink}."
```

Comment:
I'd change this text to "WMS is an [OGC standard](#) for map image services." 

Users understand that a link will take them to more information so we don't need to preface that and, this way, we provide a unique text label for the link for SR users tabbing through the page or for people looking at a rotor of link options.

Reply:
This makes sense. Done!